docs/src/examples.rst

```diff
@@ -0,0 +1,65 @@
+.. _examples:
+
+Examples
+========
+
+This page lists a few examples of visualizations based on chemiscope. 
+Rather than providing a demonstration of a scientifc application, as for the 
+examples on `the main page <http://chemiscope.org>`_, these examples
+consist in short Python scripts that demonstrate how to make a chemiscope
+viewer file that uses some particular features (e.g. atom coloring or shape
+visualization). 
+
+
+.. raw:: html
+
+    <h2>Trajectory data</h2>
+
+.. figure:: img/example_pca.png
+    :align: center
+
+    Trajectory data from a MD simulation of allyl alcohol, including 
```

Comment:
This PCA is very sparse -- it might be good to add some description of what's going on, especially because it's the first example.

Reply:
Actually I was thinking of adding a truly basic example, load-from-ase-and-visualize as the first example. Would that be fine with you?

Reply:
Added a "simple example" does it help?